var/spack/repos/builtin.mock/packages/nosource-bundle/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class NosourceBundle(BundlePackage):
+    """Simple bundle package with one dependency"""
+
+    homepage = "http://www.example.com"
+
+    version('1.0')
+
+    depends_on('dependency-install')
```

Comment:
Why add a new mock package?  This is identical to the existing `nosource` package.

Reply:
Okay, `was` identical.